### PR TITLE
Updated README expansion title

### LINKED ISSUE #353: Readme Quick Start: Consider making all bulleted titles to be fully executable
### Docs Update Description

In the main README.md, Quick Start steps 2 and 4 have:

"Results of `fidesctl evaluate`" 
and
"Run `fidesctl evaluate` again" as the bulleted titles to the expandable explanatory text.

This issue is to suggest using the full command in these bullets:

"Results of `fidesctl evaluate demo_resources/`" and "Run `fidesctl evaluate demo_resources/`"

This tracks with step 3, which includes the complete step: "Run `cat demo_resources/demo_policy.yml`"

To avoid too much repetition, 2. could begin, "Run `fidesctl evaluate` on the  `demo_resources/` folder." This keeps each step's explanatory paragraph focused on what-is-to-be-done, and allows all of the bulleted titles to be executed successfully in order.

### Additional context

Suggesting this because in step 4 I copy-pasted `fidesctl evaluate` alone and got unexpected green output that looked like success.

### PR DESCRIPTION
**Pull request by iamkelllly**
Closes #353 

### Code Changes

* [ ] updated README.md with `<summary>` tag update in step 4
* [ ] updated README.md step 2

### Steps to Confirm

* [ ] read the README for flow and reduced repetition
* [ ] read the README for consistency

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [ ] Documentation Updated

### Description Of Changes

updating README with feedback from @banagale to reduce repetition and increase readability

### CODE REVIEW

**Review by banagale — CHANGES_REQUESTED** [review]
This is good a few other suggestions:

On line 77, consider:

```
2. Use the <code>evaluate</code> command to test if this project's demo analytics and demo marketing systems are compliant with your privacy policy as code:
```

On line 80, consider:

```
<summary>Run <code>fidesctl evaluate demo_resources/</code></summary>
```

Remove line 83, given other output examples do not include the command line.

**Comment by ThomasLaPiana** [comment]
@iamkelllly anything I can do to help get this merged?

**Comment by iamkelllly** [comment]
@banagale re-requested your review here, also ccing @conceptualshark to review as well!

File: README.md [hunk 1]
```diff
@@ -171,9 +171,9 @@ Fides (*fee-dhez*, Latin: Fidēs) is an open-source tool that allows you to easi
 
     <details>
 
-      <summary>Run <code>fidesctl evaluate</code> again</summary>
+      <summary>Run <code>fidesctl evaluate demo_resources/</code> </summary>
 
-        Re-run `fidesctl evaluate demo_resources` which will cause an evaluation failure! This is because your privacy policy has 1 rule that should fail if any system uses contact information for marketing purposes, and you've just updated your marketing system to start using contact information for marketing purposes.
+      Re-run `fidesctl evaluate demo_resources` which will cause an evaluation failure! This is because your privacy policy has 1 rule that should fail if any system uses contact information for marketing purposes, and you've just updated your marketing system to start using contact information for marketing purposes.
```
Comment (conceptualshark) [inline README.md:176]: I might simplify/clean up this second sentence a little for clarity - 

Running `fidesctl evaluate demo_resources` now causes an evaluation failure. Your privacy policy's 'Reject Direct Marketing' rule disallows collecting contact information for marketing purposes, and flagged adding this data category and use to your `privacy_declarations`.

**Review by conceptualshark — APPROVED** [review]
👍 good to go with the suggested edit or not, think these changes definitely help with clarity

**Comment by iamkelllly** [comment]
Thank you for opening this issue and the readability review @banagale - Looking forward to more contributions from you :)

**Pull request merged**